client/proxy/proxy.go

```diff
@@ -161,5 +184,11 @@ func (m *ManagerClient) doConfigCRUD(key model.Key, method string, inBody []byte
 	if err != nil {
 		return nil, err
 	}
-	return m.do(request)
+
+	response, err := m.do(request)
+
+	// Log after the call to m.do() so that the full hostname is present
```

Comment:
(optional)

```golang
// Log after the call to m.do() so that the full hostname is present
defer glog.V(2).Infof("%s", m.toCurl(request, string(inBody)))
return m.do(request)
```